sup.rb

```diff
@@ -0,0 +1,41 @@
+require "language/go"
```

Comment:
Yes, will update it.

I actually open a PR on homebrew https://github.com/Homebrew/homebrew/pull/48230. 

Later on, we probably should have some automation process to update and send PRs to homebrew use the `sup.rb` version on this repository as the source.